### DISCUSSION TITLE
rustdesk.com is our Only Official Domain

### DISCUSSION THREAD

**Discussion by rustdesk** [rustdesk/rustdesk#9679] [Announcements]
Recently, we have noticed many pirate RustDesk websites, such as `rustdesk.io` and `rustdesk.pl`. These sites provide unsafe RustDesk downloads, especially `rustdesk.io`, which even appears on the first page of Google search results. We have already reported this to Google and hope more people will report it as well, so that we can urge Google to take action. We reiterate that RustDesk only has `rustdesk.com` as its official domain.

![image](https://github.com/user-attachments/assets/e863f63b-47eb-4868-b7e2-81f37a8acc37)

**Comment by Martinligabue** [comment]
not sure how much it'll help, still added to urlscan:

- https://urlscan.io/result/85c5441f-7f8a-4a5c-a687-5c320b10e49a/
- https://urlscan.io/result/5b79d9d5-dba0-4436-a727-4c186fa83c9f/

**Comment by droolio** [comment]
Reported to Google. Looks like they're serving a legit binary but with a modified filename in the .exe. I assume this is redirecting traffic to their own servers, and/or encrypting with their own API key(?).

**Comment by diggajupadhyay** [comment]
reported.

**Comment by MELERIX** [comment]
you can report all these fake webites to:

Google: https://safebrowsing.google.com/safebrowsing/report_phish/

Microsoft: https://www.microsoft.com/wdsi/support/report-unsafe-site/

VirusTotal: https://www.virustotal.com/gui/home/url

**Reply by xlionjuan** [reply]
Cloudflare Radar: https://radar.cloudflare.com/

**Comment by olumolu** [comment]
WEhat about this
https://rustdesk.co.nz/

**Comment by olumolu** [comment]
rustdesk.io is proving a non malicious binay

**Reply by Zocker1999NET** [reply]
I confirm that their Windows binary is the same as the one here from Github, but RustDesk has a feature (which IMO is handy for IT support cases) that these scammers leverage: They configure their own malicious, at least untrusted, RustDesk server via the filename of the executable, read [here](https://rustdesk.com/docs/en/self-host/client-configuration/#5-put-config-in-rustdeskexe-file-name-windows-only).

And I doubt that they just want to provide another trustworthy public RustDesk server for the public because:
- Their site only barely mentions that they are providing a server different from the "official" RustDesk servers.
- They do not explain or link back to the original RustDesk developers (i.e. this GitHub repo or rustdesk.com). Rather they link to rustdesk.pl.
- None of their links points to useful documentation of RustDesk (probably because that is hosted on rustdesk.com as well).

**Comment by Zocker1999NET** [comment]
to report another page which also resembles an older version of rustdesk.com & contains a modified download link (& many broken ones): rustdesk.secure-box.de

**Comment by hagezi** [comment]
You can also report further domains [here](https://github.com/hagezi/dns-blocklists/issues/4009). I will add them to my lists. The known domains have already been added and will be added to the lists with the next release.

Cheers,
Gerd

**Reply by rustdesk** [reply]
Thanks

**Comment by muescha** [comment]
what about `Elif Rustdesk` at Google Play Store?

**Reply by xlionjuan** [reply]
By OBS and Fedora dramas, I learned this:

https://github.com/rustdesk/rustdesk/blob/8f545491a284b15fecd6819e9e7081c9ce4a52db/LICENCE#L367

**Reply by olumolu** [reply]
Its available in fdroid
